### ISSUE TITLE
[WCAG] Nav + Hero image/text contrast doesn't meet standards

### ISSUE THREAD

**Issue by arielkidwell** [vigetlabs/pointless-corp-astro#15]
**Details**
- Both navigation and hero copy overlay doesn't meet contrast standard
- This also occurs on project detail pages with lighter background colors

![Image](https://github.com/user-attachments/assets/824abdb8-0aa8-48a7-bfb6-58624d1cae96)

**Comment by ten1seven** [comment]
@arielkidwell I've implemented programmatic color testing for project pages so the logo and nav will be black or white depending on what's needed to achieve the best contrast:

![Image](https://github.com/user-attachments/assets/846bb7dd-cbd0-432b-b588-3a46e5ab8919)

![Image](https://github.com/user-attachments/assets/be137c26-ff82-4e8a-b727-5a881e61d3f3)

The home page is both a case of the text over an image rather than a solid color and the white being iffy contrast with the image. I added a subtle 40% to 0% black gradient behind the logo/nav on the home page to boost the contrast. Grabbing pixel values in the photo around the nav text now gives me passing contrast values.

**Comment by arielkidwell** [comment]
@ten1seven Noticed this while I was testing and can confirm it works great!
